src/stacks/nestedLogGroup.ts

```diff
@@ -63,17 +48,15 @@ export class NestedLogGroupAlarmsStack extends BaseNestedStack {
     groupName: string,
     metricFilterName: string,
     pattern: string,
-    localConf: config.ConfigMetricAlarms,
+    localConf: config.ConfigMetricAlarm,
   ): void {
-    const isEnabled = config.configIsEnabled(this.defaultType, metricFilterName, localConf);
-    const suffix = config.configAlarmSuffix(this.defaultType, metricFilterName, localConf);
-    const fullMetricFilterName = suffix ? `${metricFilterName}-${suffix}` : metricFilterName;
-
-    if (!isEnabled) {
+    const isEnabled = localConf.enabled !== false;
+    const localEnabled = Object.values(localConf.alarm || {}).find(l => l.enabled === true);
```

Comment:
This could be extracted as library code, it's being used in at least two different places.

Reply:
Will do actually